importer/src/map_config.rs

```diff
@@ -0,0 +1,74 @@
+use abstio::{CityName, MapName};
+use geom::Distance;
+use map_model::DrivingSide;
+
+/// Given the name of a map, configure its import.
```

Comment:
All of those deleted config files are now generated by this one bit of code. I feel like this is way simpler